server/meeting.go

```diff
@@ -75,9 +75,18 @@ func (p *Plugin) GenerateHashtag(channelID string, nextWeek bool, weekday int) (
 			return "", err
 		}
 	} else {
-		// Get date for the list of days of the week
-		if meetingDate, err = nextWeekdayDateInWeek(meeting.Schedule, nextWeek); err != nil {
-			return "", err
+		// user didn't provide any specific date, Get date for the list of days of the week
+		if !requeue {
+			if meetingDate, err = nextWeekdayDateInWeek(meeting.Schedule, nextWeek); err != nil {
+				return "", err
+			}
+		} else {
+			if len(meeting.Schedule) == 1 && meeting.Schedule[0] == assignedDay { // if this day is the only day selected in settings
```

Comment:
0/5 It's better to put the comment on the next line since the line is already pretty long

```suggestion
			if len(meeting.Schedule) == 1 && meeting.Schedule[0] == assignedDay {
			// if this day is the only day selected in settings
```

Reply:
done